pkgs/by-name/wa/waydroid-script/package.nix

```
@@ -0,0 +1,57 @@
+{
+  python3,
+  waydroid,
+
+  fetchFromGitHub,
+  stdenv,
+  lib,
+  makeWrapper,
+}:
+stdenv.mkDerivation rec {
+  name = "waydroid-script";
+  version = "unstable-2025-07-13";
```

```suggestion
  version = "0-unstable-2025-07-13";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning